inflate.c

```diff
@@ -15,7 +15,7 @@
 
 /* function prototypes */
 static int inflateStateCheck(PREFIX3(stream) *strm);
-static int updatewindow(PREFIX3(stream) *strm, const uint8_t *end, uint32_t copy);
+static int updatewindow(PREFIX3(stream) *strm, const uint8_t *end, uint32_t len, int copy);
```

Comment:
It would be good to use fixed width type `int32_t`.

Reply:
Sure.  I had just been matching the type defined in inflate_state.